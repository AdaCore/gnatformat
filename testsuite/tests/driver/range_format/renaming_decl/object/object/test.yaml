description: ObjectDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 44",
    "-EL 45",
    "--pipe",
  ]
