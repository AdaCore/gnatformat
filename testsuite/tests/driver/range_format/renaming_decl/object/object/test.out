test.ads(<ObjectDecl ["Renamed_Head"] test.ads:44:4-44:37>) - 44:1-44:37
^
   Renamed_Head : Link renames Head;$

